frontend/src/MyProfile/ProfileModal.jsx

```diff
@@ -78,7 +103,28 @@ export default function ProfileModal({ isOpen, onClose, handleSubmitPatientInfo,
           <input name="occupation" placeholder="Occupation" value={formData.occupation} onChange={handleChange} required />
           <input name="address" placeholder="Address" value={formData.address} onChange={handleChange} required />
           <input name="phone" placeholder="Phone" type="tel" value={formData.phone} onChange={handleChange} required />
-          <textarea name="complaint" placeholder="Complaint (optional)" value={formData.complaint} onChange={handleChange} />
+
+          <label className="label">Symptoms (1 = least important, 5 = most important)</label>
```

Comment:
looks good so far! i think if the symptoms and prescription section gets longer/requires more to maintain, we can probably have separate components for them.